### ISSUE TITLE
Problem with google analytics

### ISSUE THREAD

**Issue by tommasoflorio** [litespeedtech/lscache_wp#375]
With the plugin actived analytics do not work properly, seems the total visit count is wrong and also the source of the visit seems are all direct visit. when i disable the plugin analytics comeback to normal behaviour

**Comment by bubobih** [comment]
Disable guest mode

**Comment by tommasoflorio** [comment]
Yes, but with this setting i will lose the good google pagespeed score. There some other suggestions?

**Comment by bubobih** [comment]
add gtag to Guest Mode JS Excludes  in ?page=litespeed-page_optm#settings_tuning

**Comment by tommasoflorio** [comment]
Already done but it not working.

**Comment by bubobih** [comment]
for me its working ok

**Comment by hi-hai** [comment]
@tommasoflorio Please try v4.2-rc2+. There was a bug in v4.1 if the Guest Mode UA setting is missing which will cause failed to detect visitors. You can also check if your GM UA setting is there or not. If not, you can either upgrade or add back the default UA setting values:
```
Lighthouse
GTmetrix
Google
Pingdom
bot
PTST
HeadlessChrome
```
